src/components/TreeList/__stories__/TreeList.mdx

```diff
@@ -0,0 +1,312 @@
+### multiple
+
+This prop is necessary for the correct view of the selected elements since the state of the selected elements is controlled from the parent component.
+
+### setActiveItemId
+
+Required for correct keyboard interactions. While navigating through the keyboard, you need to set the next active element.
+
+### defaultGroupsExpanded
+
+Control the default expanded state of items' groups. Default - `true`.
+
+### renderItem
+
+The ability to completely redefine the rendering of a list item. For example, add dividers between list items or wrap an item in a link component. As a view component to display a list item, use [ListItemView](/docs/unstable-uselist--docs#listitemview);
+
+```tsx
+<TreeList
+    renderItem={({
+        data,
+        props,
+        index,
+        renderContainerProps,
+        itemState: {groupState, isLastItem, itemState},
+    }) => {
+        return <ListItemView {...props} {...renderContainerProps} />;
+    }}
+/>
+```
+
+#### renderItem function argument object:
+
+-   `data` - access to the original object with the data of the sheet element;
+-   `props` - default props generated by the component taking into account the state (whether the element is selected or not, active, disclosed). The set of returned passes corresponds to the result of the function execution [getItemRenderState](/docs/unstable-uselist--docs#item-state-props);
+-   `index` - ordinal index of the element, taking into account that with a tree-like data structure, the list elements have a flatten representation;
+-   `renderContainerProps` - the passes thrown from the redefined container. In the vast majority of cases, you won't need it, but it's worth knowing that there is such a possibility.;
+-   `context` - useful information about the current list item:
+    -   `itemState` - meta info about item
+        -   `indentation` - integer number representing nested list level
+        -   `parentId` - `id` of parent list item;
+    -   `groupState` - An optional parameter. If the list item is also the first item of the nested list:
+        -   `childrenIds` - array of `id` of nested list items;
+    -   `isLastItem` - is the current item the last one in the list;
+
+> Important! Absolutely all the props for [ListItemView](/docs/unstable-uselist--docs#listitemview) can be redefined in the renderItem method. This is the preferred method for changing the view of the sheet elements.
+
+### renderContainer
+
+Ability to override default list container `ListContainerView`;
+
+```tsx
+<TreeList
+    renderContainer={({
+        id,
+        containerRef,
+        visibleFlattenIds,
+        renderItem,
+        size,
+        className,
+        qa,
+    }: TreeListRenderContainerProps<T>) => {
+        return (
+            <ListContainerView
+                qa={qa}
+                className={className}
+                fixedHeight // by default 300px. Use `--g-list-height` css custom propery to override container height if this prop active
```

Comment:
I just noticed. Why we are using CSS here instead of having `height: number` prop?

Reply:
it is a legacy from previous list. We can improve this moment later